README.md

```diff
@@ -1,55 +1,35 @@
 # iRODS TypeScript HTTP API Wrapper
 
-## Installation and Usage
-
-1. Install npm package: 
+## Installation
 
 ```
 npm install git+https://github.com/irods/irods_client_http_typescript
 ```
 
-2. Import `IrodsHttpClient` class:
+## Usage
```

Comment:
Please add words following this title describing what the code snippet is demonstrating.

The reader just needs a general idea of what it is doing.

Reply:
Done.